src/dbnode/storage/dice.go

```diff
@@ -18,25 +18,25 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-package dice
+package storage
 
 import (
 	"fmt"
 
 	"github.com/MichaelTJones/pcg"
 )
 
-// Dice is an interface that allows for random sampling.
-type Dice interface {
-	// Rate returns the sampling rate of this Dice: a number in (0.0, 1.0].
+// dice is an interface that allows for random sampling.
+type dice interface {
```

Comment:
hmm is it useful to keep this interface around?

Reply:
internal interfaces are so infrequently useful to begin with - in this case, definitely not, since we're not even mocking it afaik.

Reply:
tbh I would just s/epoch/dice/ and call it a day.

Reply:
hmm, it looks like we create a fake flavor of the interface for testing purposes (essentially mocking and asserting on # of invocations: https://github.com/m3db/m3/blob/f4c26947ba41aee0fbe7b978f0a42bab6a58e5c0/src/dbnode/storage/forward_index_dice_test.go#L114-L117

Reply:
Ah, missed that one. In that case, this is good as-is - consolidating and un-exporting is the more important goal. Thanks!